starter_code/src/movies.js

```diff
@@ -1,9 +1,29 @@
 /* eslint no-restricted-globals: 'off' */
+console.log (movies[0]);
 // Turn duration of the movies from hours to minutes 
+function stringToMinutes(string) {
+    var arr = string.split('h')
+    var hours = Number(arr[0])
+    var minutes = Number(arr[1].split("min")[0])
+
+    return hours*60+minutes
+}
+ 
+function turnHoursToMinutes(moviesarr){
+    var newTime = moviesarr.map(function(movie){
+        return movie.duration= stringToMinutes;
+    });
+}
 
 
-// Get the average of all rates with 2 decimals 
 
+// Get the average of all rates with 2 decimals 
```

Comment:
this looks good already 👍

Reply:
Well according to Jasmine I've done everything wrong so IDK.
made a new push...